src/makielayout/blocks/axis3d.jl

```diff
@@ -664,29 +664,42 @@ function add_panel!(scene, ax, dim1, dim2, dim3, limits, min3)
         string((:x, :y, :z)[dim2]) * string(sym))
     attr(sym) = getproperty(ax, dimsym(sym))
 
-    vertices = lift(limits, min3) do lims, mi3
-
+    rect = lift(limits) do lims
         mi = minimum(lims)
         ma = maximum(lims)
+        Polygon([
+            Point2(mi[dim1], mi[dim2]),
+            Point2(ma[dim1], mi[dim2]),
+            Point2(ma[dim1], ma[dim2]),
+            Point2(mi[dim1], ma[dim2])
+        ])
+    end
 
-        v3 = if mi3
-            mi[dim3] + 0.005 * (mi[dim3] - ma[dim3])
-        else
-            ma[dim3] + 0.005 * (ma[dim3] - mi[dim3])
-        end
+    plane_offset = lift(limits, min3) do lims, mi3
+        mi = minimum(lims)
+        ma = maximum(lims)
 
-        p1 = dim3point(dim1, dim2, dim3, mi[dim1], mi[dim2], v3)
-        p2 = dim3point(dim1, dim2, dim3, mi[dim1], ma[dim2], v3)
-        p3 = dim3point(dim1, dim2, dim3, ma[dim1], ma[dim2], v3)
-        p4 = dim3point(dim1, dim2, dim3, ma[dim1], mi[dim2], v3)
-        [p1, p2, p3, p4]
+        mi3 ? mi[dim3] : ma[dim3]
     end
 
-    faces = [1 2 3; 3 4 1]
+    plane = Symbol((:x, :y, :z)[dim1], (:x, :y, :z)[dim2])
 
-    panel = mesh!(scene, vertices, faces, shading = NoShading, inspectable = false,
+    panel = poly!(scene, rect, inspectable = false,
         xautolimits = false, yautolimits = false, zautolimits = false,
-        color = attr(:panelcolor), visible = attr(:panelvisible))
+        color = attr(:panelcolor), visible = attr(:panelvisible),
```

Comment:
Should we specify `strokecolor = :transparent` here, just in case we end up changing that default?

Reply:
Yes good idea